src/algebra/monoid_algebra/basic.lean

```diff
@@ -295,9 +295,8 @@ instance [monoid R] [semiring k] [distrib_mul_action R k] [has_faithful_smul R k
   has_faithful_smul R (monoid_algebra k G) :=
 finsupp.has_faithful_smul
 
-instance [monoid R] [monoid S] [semiring k] [distrib_mul_action R k] [distrib_mul_action S k]
-  [has_smul R S] [is_scalar_tower R S k] :
-  is_scalar_tower R S (monoid_algebra k G) :=
+instance [semiring k] [smul_zero_class R k] [smul_zero_class S k] [has_smul R S]
+  [is_scalar_tower R S k] : is_scalar_tower R S (monoid_algebra k G) :=
```

Comment:
I assume @Hagb missed these (and the few others) in their PR?

Reply:
Yea, guess so

Reply:
> I assume @Hagb missed these (and the few others) in their PR?

ummm... I didn't notice this before.